tests/config/settings-config.yaml

```diff
@@ -0,0 +1,2 @@
+server-settings:
+    server.thread_pool: 1000
```

Comment:
Apparently not :)

It was connected to the test I removed, in favor of the ones you wrote. Yours just create a config inline, right? If that's the case, I'll remove this file.